apps/protocol-frontend/src/components/ContributionsHeatMap.tsx

```diff
@@ -40,7 +121,7 @@ const ContributionsHeatMap = ({
             data={contributionsCountMap.filter(
               contribution => contribution.value !== 0,
             )}
-            from={subWeeks(new Date(), 52)}
+            from={subWeeks(new Date(), YEAR)}
```

Comment:
nice. nit: maybe YEAR_WEEKS?

Reply:
great call. updated that + the impacted files